ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Image/ImagePageVulnerabilities.tsx

```diff
@@ -125,10 +122,11 @@ function ImagePageVulnerabilities({
             field: 'Severity',
             direction: 'desc',
         },
-        onSort: () => setPage(1),
+        onSort: () => setPage(1, 'replace'),
```

Comment:
Unrelated, but fixes an extra history state bug.

Reply:
Could you remind me when we need to use `replace` and when we shouldn't? Looking at the recent features, we seem to use `replace` with `setPage`. Would it be better to make it a default action or does that mess anything up?

Reply:
We need to use `'replace'` on all updates to the URL state, _except the first_, whenever we are making multiple URL parameter changes in a single function call. This is because otherwise there will be multiple history states pushed to the stack which can cause bugs when the user presses the Back button.

It is easier to see in an example like:
```ts
() => {
    setSearchFilter({});
    setPage(1, 'replace');
    setSort(defaultSort, 'replace');
}
```

Unfortunately in this case it is a little difficult to see why since `useURLSort()` sets the sort parameter internally. There might be a better generalized solution that will do this automatically, but it would probably require a design doc and definitely will not be ready for 4.5.